src/main/java/me/conorthedev/mediamod/gui/PlayerOverlay.java

```diff
@@ -361,13 +372,58 @@ void drawPlayer(int cornerX, int cornerY, boolean isModern, boolean testing, dou
         GlStateManager.popMatrix();
     }
 
+    public static int getComplementaryColor(Color colorToInvert) {
+        double y = (299 * colorToInvert.getRed() + 587 * colorToInvert.getGreen() + 114 * colorToInvert.getBlue()) / 1000.0;
+        return y >= 128 ? Color.BLACK.getRGB() : Color.WHITE.getRGB();
+    }
+
+    private static String formatTime(int milliseconds) {
+        return DATE_FORMAT.format(Date.from(Instant.ofEpochMilli(milliseconds)));
+    }
+
+    public static void drawRect(double left, double top, double right, double bottom, int color)
+    {
```

Comment:
Why is the code formatting not consistent?

Reply:
because intellij didnt do its job correctly aaaaa